src/development/ssh.md

```diff
@@ -71,6 +71,20 @@ You have your SSH keys (if not, take a look at the section above), but you need
 
 That's it! You're all set. Now you'll be able to use Git and command shells with any Platform.sh environment that your user account is authorized to work with.
 
+### Forwarding keys by default
```

Comment:
Is this still necessary with the integrated Dev (wings)?  I thought it wasn't anymore.  (We only document the integrated approach, not the old legacy one.)

Reply:
Not sure I understand. Integrated approach? Using the CLI?

Reply:
`platform ssh -e production` gets you to the prod environment these days on Dedicated.  So, when would someone still need to setup key forwarding?

Reply:
I was assuming for rsync/scp